src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -4,40 +4,19 @@
 using System;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
-using static Microsoft.Data.Sqlite.Interop.Constants;
-
-#if NET451 || DOTNET5_4
-using System.IO;
-using System.Reflection;
-using Microsoft.Extensions.PlatformAbstractions;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Internal;
+
+#if NET451
+using Microsoft.Data.Sqlite.Utilities;
 #endif
 
 namespace Microsoft.Data.Sqlite.Interop
 {
     internal static class NativeMethods
     {
-#if NET451 || DOTNET5_4
+#if NET451
         static NativeMethods()
         {
             var loaded = NativeLibraryLoader.TryLoad("sqlite3");
```

Comment:
Ok, that’s what I was missing. Can we add into the comments?The MSBuild fixes this for full .NET when platform arch is known, right?